capstone_project/src/main/java/com/google/sps/servlets/MapServlet.java

```diff
@@ -0,0 +1,79 @@
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.repackaged.com.google.gson.Gson;
+import com.google.sps.data.School;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that handles loading school data for map markers. */
+@WebServlet("/school-data")
+public class MapServlet extends HttpServlet {
+
+  private ArrayList<School> schools = new ArrayList<School>();
+
+  private void addUCIToSchools() {
+    ArrayList<String> uciNews =
+        new ArrayList<String>(
+            Arrays.asList(
+                "https://news.uci.edu/2020/06/30/virtual-nurses-make-a-real-difference/",
+                "https://news.uci.edu/2020/07/07/uci-chancellor-emeritus-michael-v-drake-named-university-of-california-president/",
+                "https://news.uci.edu/2020/06/18/uci-podcast-jessica-millward-on-the-meaning-and-importance-of-juneteenth/"));
+    School uci = new School("UCI", 33.640339, -117.844248, uciNews);
+    schools.add(uci);
+  }
+
+  private void addUWToSchools() {
+    ArrayList<String> uwNews =
+        new ArrayList<String>(
+            Arrays.asList(
+                "https://www.washington.edu/coronavirus/2020/07/10/back-to-school-town-hall/",
+                "https://www.washington.edu/news/2020/07/16/wsas-2020/?utm_source=UW%20News&utm_medium=tile&utm_campaign=UW%20NEWS",
+                "https://www.washington.edu/news/2020/07/29/expert-faq-wildfires-in-the-pacific-northwest-during-the-covid-19-pandemic/"));
+    School uw = new School("UW", 47.655277, -122.303606, uwNews);
+    schools.add(uw);
+  }
+
+  private void addUWBToSchools() {
+    ArrayList<String> uwbNews =
+        new ArrayList<String>(
+            Arrays.asList(
+                "https://news.uci.edu/2020/06/30/virtual-nurses-make-a-real-difference/",
+                "https://news.uci.edu/2020/07/07/uci-chancellor-emeritus-michael-v-drake-named-university-of-california-president/",
+                "https://news.uci.edu/2020/06/18/uci-podcast-jessica-millward-on-the-meaning-and-importance-of-juneteenth/"));
+    School uwb = new School("UWB", 47.758821, -122.190725, uwbNews);
+    schools.add(uwb);
+  }
+
+  /*Add some hard-coded School objects to the array of schools that will later be converted into markers on the map.*/
+  public MapServlet() {
+    addUCIToSchools();
+    addUWToSchools();
+    addUWBToSchools();
+  }
+
```

Comment:
This is up to you, so you don't have to do this but I recommend moving these into their own class. Since you have a School class and these are hard coded, maybe you could move these into the constructor of the School for now. Like a Hashmap with the key being the school name and the value being an ArrayList of Strings. Again, you don't have to do this